src/utils/data/getPropertyValues.js

```diff
@@ -0,0 +1,26 @@
+/*
+ * @flow
+ */
+
+import { Map, set } from 'immutable';
+import { Models } from 'lattice';
+
+import getPropertyValue from './getPropertyValue';
+
+const { FQN } = Models;
+
+export default function getPropertyValues(
+  entity :Map | Object,
+  fqns :Array<FQN | string>,
+  fallback ?:boolean | number | string = '',
+) :Object {
```

Comment:
I'm wondering whether the existence of this function is actually necessary... In your crc pr, you have:

```
const [firstName, lastName, dob] = getPropertyValues(person, [
  GIVEN_NAME,
  SURNAME,
  DOB
]);
```

Why can't you just call `getPropertyValue(entity, fqn)` for each property? It would be the exact same thing since `getPropertyValues` is just a for-loop. Additionally, what if you wanted a different `fallback` depending on the property? Well, in that case, it feels a bit cumbersome to have `getPropertyValues` also accept a `fallback` parameter per property, i.e.

```
const [firstName, lastName, dob] = getPropertyValues(
  person, 
  [GIVEN_NAME, SURNAME, DOB],
  ['---', '---', '']
);

// vs...

const firstName = getPropertyValue(person, GIVEN_NAME, '---');
const lastName = getPropertyValue(person, SURNAME, '---');
const dob = getPropertyValue(person, DOB, '');
```

What do you think?

Reply:
That's true that `getPropertyValues` is essentially just a for-loop, whether the result is an array or an object. Adding a fallback property to `getPropertyValues` that allows for a different fallback value for each given property, which @anzioka also brought up, does feel a bit cumbersome, and so maybe it's better to just call `getPropertyValue` for each property in a situation where you need different fallback values/types. I would be in favor of just removing `getPropertyValues`.